sailor/sap_iot/fetch.py

```diff
@@ -217,6 +227,7 @@ def get_indicator_data(start_date: Union[str, pd.Timestamp, datetime.timestamp,
     results = pd.DataFrame(columns=schema.keys()).astype(schema)
 
     print('Waiting for data export:')
+    start_time = time.time()
```

Comment:
you probably want `time.monotonic` rather than `time.time` here (and below) to avoid issues with things like daylight savings

Reply:
The unix UTC timestamp does not contain any timezone or DST information, so `time.time()` is fine.

But `time.monotonic()` is still the better choice here since we don't really need a timestamps, we only need a clock :)

Reply:
`time.time()` changed to `time.monotonic()`